packages/govuk-frontend/src/govuk/components/accordion/accordion.yaml

```
@@ -48,30 +48,45 @@ params:
     required: true
     description: An array of sections within the accordion.
     params:
-      - name: heading.text
-        type: string
+      - name: heading
+        type: object
         required: true
-        description: The title of each section. If `heading.html` is supplied, this is ignored.
-      - name: heading.html
-        type: string
-        required: true
-        description: The HTML content of the header for each section. Used as the title for each section. The header is inside the HTML `<button>` element, so you can only add [phrasing content](https://html.spec.whatwg.org/#phrasing-content) to it.
-      - name: summary.text
-        type: string
-        required: false
-        description: Text content for summary line. If `summary.html` is supplied, this is ignored.
-      - name: summary.html
-        type: string
+        description: The heading of each section.
+        params:
+          - name: text
+            type: string
+            required: true
+            description: If `html` is set, this is not required. The heading text of each section. If `html` is provided, the `text` option will be ignored.
+          - name: html
+            type: string
+            required: true
+            description: If `text` is set, this is not required. The heading HTML content of each section. The header is inside the HTML `<button>` element, so you can only add [phrasing content](https://html.spec.whatwg.org/#phrasing-content) to it. If `html` is provided, the `text` option will be ignored.
```

To mirror the other suggestion:

```suggestion
            description: The heading HTML content of each section. The header is inside the HTML `<button>` element, so you can only add [phrasing content](https://html.spec.whatwg.org/#phrasing-content) to it. If `html` is provided, the `text` option will be ignored. If `text` is provided, this is not required.
```

---

It's a good suggestion, but we have **74 results in 27 files** searching for "option will be ignored" 🤦‍♂️